src/backend/base/langflow/alembic/versions/5ace73a7f223_new_remove_table_upgrade_op.py

```diff
@@ -0,0 +1,62 @@
+"""new remove table upgrade op
+
+Revision ID: 5ace73a7f223
+Revises: 0ae3a2674f32
+Create Date: 2024-10-08 10:59:12.980671
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel
+from sqlalchemy.engine.reflection import Inspector
+from langflow.utils import migration
+from sqlalchemy.dialects import sqlite
+from langflow.utils import migration
+
+# revision identifiers, used by Alembic.
+revision: str = "5ace73a7f223"
+down_revision: Union[str, None] = "0ae3a2674f32"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    conn = op.get_bind()
+    # ### commands auto generated by Alembic - please adjust! ###
+    if migration.table_exists("vertex_build", conn):
+        op.drop_table("vertex_build")
+
+    with op.batch_alter_table("message", schema=None) as batch_op:
+        batch_op.alter_column("text", existing_type=sa.TEXT(), nullable=True)
+
+    # ### end Alembic commands ###
+
+
+def downgrade() -> None:
+    conn = op.get_bind()
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("message", schema=None) as batch_op:
+        batch_op.alter_column("text", existing_type=sa.TEXT(), nullable=False)
+
+    if not migration.table_exists("vertex_build", conn):
+        op.create_table(
+            "vertex_build",
+            sa.Column("timestamp", sa.DATETIME(), nullable=False),
+            sa.Column("id", sa.VARCHAR(), nullable=False),
+            sa.Column("data", sqlite.JSON(), nullable=True),
+            sa.Column("artifacts", sqlite.JSON(), nullable=True),
+            sa.Column("params", sa.TEXT(), nullable=True),
+            sa.Column("valid", sa.BOOLEAN(), nullable=False),
+            sa.Column("flow_id", sa.CHAR(length=32), nullable=False),
+            sa.Column("build_id", sa.CHAR(length=32), nullable=False),
+            sa.ForeignKeyConstraint(
+                ["flow_id"],
+                ["flow.id"],
+            ),
+            sa.PrimaryKeyConstraint("build_id"),
+        )
```

Comment:
```suggestion

```

Reply:
same here with the downgrade we might need to add a vertex_build table.

Reply:
Removing it passes the test, but I guess those tests are dependent on the vertex build table.

Reply:
Removed the check related to vertex_build:

But the alembic check fails even though pytest passes and langflow run successfully.

@jordanrfrazier @ogabrielluiz